lib/realtime/tenants/connect.ex

```diff
@@ -164,6 +185,26 @@ defmodule Realtime.Tenants.Connect do
     {:noreply, state}
   end
 
+  def handle_info(
+        {:notification, _, _, "realtime:broadcast", payload},
```

Comment:
Feels like this is odd in the connect module. Too much responsibility for it, but nit for now.

Reply:
agreed, will try to check what I can do to move this out of connect (start the db conn here and move the listen somewhere else 🤔 )